src/hooks/carousel/__tests__/useSingleCarousel.browser.spec.js

```diff
@@ -0,0 +1,67 @@
+import '&test/browser/styles/main.css';
+import React, { useRef, useState } from 'react';
+import { Box } from '&test/browser/utils/Box';
+import { mountHookWithUI } from '&test/browser/utils/mountHookWithUI';
+import { useSingleCarousel } from '../useSingleCarousel';
+
+const mainStyle = `
+  .list {
+    width: 400px;
+  }
+`;
+
+const useWrappedHook = (result) => {
+  const ref = useRef();
+  const [selected, setSelected] = useState(0);
+  result.current = { selected, setSelected };
+
+  const { props: listProps } = useSingleCarousel(ref, { selected, onSelect: setSelected, numTiles: 3 });
+
+  return (
+    <div className="list" ref={ref} {...listProps}>
+      <Box id="box-1"><button type="button">yolo</button></Box>
+      <Box id="box-2"><button type="button">yolo</button></Box>
+      <Box id="box-3"><button type="button">yolo</button></Box>
+      <div className="margin-fix" data-lwc-ignore />
+    </div>
+  );
+};
+
+const mountTest = (props) =>
+  mountHookWithUI(useWrappedHook, props, mainStyle);
```

Comment:
I like this pattern with the `useWrappedHook` to test the hook! It's very readable 👏

Reply:
Thanks! :fire: I tried really hard to make it readable & easy to use.